build/set-env.ps1

```diff
@@ -21,6 +21,8 @@ if ($Env:ENABLE_QIRRUNTIME -ne "false" -and $Env:QIR_DROPS -eq $null) {
 
 If ($Env:DROPS_DIR -eq $null) { $Env:DROPS_DIR =  [IO.Path]::GetFullPath((Join-Path $PSScriptRoot "..\drops")) }
 
+If ($Env:TOOLS_OUTDIR -eq $null) { $Env:TOOLS_OUTDIR =  (Join-Path $Env:DROPS_DIR "internal_nugets") }
```

Comment:
Let's rename the variable to `INTERNAL_TOOLS_OUTDIR`, and let's name the actual folder `internal_tools` as we might want to put tools other than nuget packages in there.

Reply:
done